content/doc/book/managing/groovy-hook-scripts.adoc

```diff
@@ -51,6 +51,8 @@ Jenkins.instance.doQuietDown();
 Output is logged to the Jenkins log file. For Debian based users, this
 is /var/log/jenkins/jenkins.log
 
+NOTE: If using the Jenkins Docker image, `$JENKINS_HOME/init.groovy.d/` does not exist.
+Instead, you should place the Groovy script files in `/usr/share/jenkins/ref/init.groovy.d/`, as this directory is copied when the Docker container starts.
```

Comment:
WDYT about pointing to the official explanation https://github.com/jenkinsci/docker?tab=readme-ov-file#installing-more-tools so user has a clue on the "why" ?

Reply:
sure Sir @dduportal ;will do it.

Reply:
hello @dduportal Sir i have updated and included it in line 56.hope it completes the line now;if further inclusion or changes please let me know.thanks